open_humans/views.py

```diff
@@ -33,6 +33,7 @@
 from .forms import ActivityMessageForm
 from .mixins import SourcesContextMixin
 from .models import BlogPost, Member, GrantProject
+from .settings import LOGIN_URL
```

Comment:
also here
https://docs.djangoproject.com/en/2.1/topics/settings/#using-settings-in-python-code